### ISSUE TITLE
Can not upload image

### ISSUE THREAD

**Issue by MaxCCC** [TryGhost/Ghost#9295]
### Issue Summary

A summary of the issue and the browser/OS environment in which it occurs.
Can not upload image (button "upload image" isn't doing anything)

### Steps to Reproduce
Can not upload image (button "upload image" isn't doing anything)

Any other info e.g. Why do you consider this to be a bug? What did you expect to happen instead?

### Technical details:

* Ghost Version: 1.3.0
* Node Version: v6.12.0
* Browser/OS: Ubuntu 16.04 (every browser)
* Database: MySQL

It's not working for any template (also not the default casper 2.1.7)

**Comment by acburdine** [comment]
Closing a duplicate of #9292

**Comment by acburdine** [comment]
@MaxCCC if you upload to the latest version of Ghost (1.18.2) this should be fixed.

**Comment by phyrexian** [comment]
Hey @acburdine 

This does not appear to be fixed in 1.18.2

```
=> ghost version
Ghost-CLI version: 1.4.0
Ghost Version (at /home/username/repos/www.site.tld): 1.18.2
```

Steps to reproduce; 

* follow https://docs.ghost.org/v1/docs/install and do an SQLITE install
* create a new story
* click the "Upload Image(s)" button in the editor footer
* Pick a file from the (in my case osx system) filepicker and click "open"

Nothing happens;  chrome/firefox (latest of each) no request is made to the server; (network-tab in chrome console / no errors on the JS console / etc)

IE: http://recordit.co/zFAI59CM7V.gif

Retried the installation;  tried updating (latest available is 1.18.2) etc.  Looks (to me) like the shortcut doesn't fire the event.

**Comment by kevinansfield** [comment]
This issue was incorrectly closed because the original issue description did not specify enough information for it not to be seen as a duplicate. https://github.com/TryGhost/Ghost/issues/9300 is now tracking the issue described [here](https://github.com/TryGhost/Ghost/issues/9295#issuecomment-348670589).
